src/components/ForexRatesTable.js

```diff
@@ -7,6 +7,17 @@ import { uid } from 'react-uid';
 
 import TablePaginationActions from './TablePaginationActions';
 
+export const onlyCurrency = (forexRates, currencyPair) => {
+  if (!currencyPair) {
+    return forexRates;
+  }
+  return forexRates.filter((rate) => rate.currencyPair === currencyPair);
+};
+
+export const setForexRatesOnly = (
+  setForexRates, currencyPair, onlyCurrencyFunc = onlyCurrency,
+) => (newRates) => setForexRates(onlyCurrencyFunc(newRates, currencyPair));
```

Comment:
This doesn't really look like it's used anywhere except the test? Do we need it?

Reply:
I can get rid of it, it was part of an earlier attempt.